int32 nodeID
bool status

# nodeID: tsl_customer = 1, tailsection = 2, altimeter = 3, GPS = 4, compass = 5, SMS_modem = 6